templates/eap-cd-image-stream.json

```diff
@@ -306,6 +349,81 @@
                     }
                 ]
             }
+        },
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "eap-cd-runtime-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP continuous delivery runtime image",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "18.0"
+                }
+            },
+            "labels": {
+                "xpaas": "18.0"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
+                        "annotations": {
+                            "description": "The latest available build of JBoss EAP continuous delivery runtime image [Tech Preview]",
+                            "iconClass": "icon-eap",
+                            "tags": "eap,javaee,java,jboss,hidden",
+                            "supports": "eap:7.3,javaee:8,java:8",
+                            "version": "latest",
+                            "openshift.io/display-name": "JBoss EAP continuous delivery (Tech Preview)"
```

Comment:
Should runtime image be mentioned there?

Reply:
I think we should be careful about changing these at this point, updating java:8->11 is fine, adding runtime, not so sure. In any case they're all marked hidden, so this is something we can update later.